src/main/resources/api.yaml

```diff
@@ -585,22 +585,28 @@ definitions:
   SprintSummaryDto:
     type: object
     properties:
+      sprint:
```

Comment:
Ten sprint nam tutaj na co? Jego już się pozyskuje z zapytania z idkiem sprintu w sumie. Znaczy nie wiem, czy tak jest dobrze, czy tak jest źle, ale pytam o motywację

Reply:
Chociaż w sumie tu nie było chyba żadnej identyfikacji sprintu, to nie wiem, czy tak jest dobrze

Reply:
ze sprintu potrzebujemy 5 pol zawsze i i tak potrzebujemy tego do liczenia na backendzie przy tworzeniu summary, lepiej przeslac to od razu niz czekac na drugie zapytanie

Reply:
Aaa, bo daty + trzy liczby są w sprincie, w sumie spoko